spec/spec_helper.rb

```diff
@@ -1,5 +1,7 @@
+require 'fakeredis'
 require 'resque'
 require 'resqutils'
+require 'resqutils/spec'
```

Comment:
Must require custom spec helpers for `spec/resque_matchers_spec.rb` to [`include Resqutils::Spec::ResqueHelpers`](https://github.com/stitchfix/resqutils/pull/6/files#diff-ebad8b958c9407c2b4f044c43486591cR5).